js/components/user-bar.js

```diff
@@ -48,6 +57,10 @@ define([
 			this.updateJobStatus = this.updateJobStatus.bind(this);
 			this.clearJobNotifications = this.clearJobNotifications.bind(this);
 			this.clearJobNotificationsPending = this.clearJobNotificationsPending.bind(this);
+
+			if (!appConfig.userAuthenticationEnabled) {
+				this.startPolling();

```

Comment:
@anthonysena / @chrisknoll , how is it supposed to work in secured app mode?

Reply:
The subscription here: https://github.com/OHDSI/Atlas/pull/861/files#diff-704b7a14b7c7b575513079f09614f96dR39 will handle the situation where security is enabled. The bug that this change addresses is that we were not polling for updates when security is disabled. When security is enabled, the subscription will start polling for updates once the user is logged in.

Reply:
@anthonysena , ouch, my bad - will be more attentive next time. Thanks for pointing me